docs/self-managed/components/management-identity/configuration/connect-to-an-oidc-provider.md

```diff
@@ -169,7 +138,7 @@ Ensure you register a new application for each component.
 You can connect to your OIDC provider through either environment variables or Helm values. Ensure only one configuration option is used.
 :::
 
-<Tabs groupId="optionsType" defaultValue="env" queryString values={[{label: 'Environment variables', value: 'env' },{label: 'Helm values', value: 'helm' }]} >
+<Tabs groupId="optionsType" defaultValue="env" queryString values={[{label: 'Environment variables', value: 'env' }]} >
```

Comment:
We could add a tab `Application.yaml` here that gives the yaml-formatted application config (to make it consistent with how we have it in the Orchestration Cluster config guide, e.g. here: https://docs.camunda.io/docs/next/self-managed/components/orchestration-cluster/identity/connect-external-identity-provider/?optionsType=yaml#step-4-configure-the-oidc-connection-details).

You can create that by translating the env vars into a hierarchical yaml structure, so roughly like:
```
camunda:
  identity:
    type: "MICROSOFT"
    baseUrl: <IDENTITY_URL>
    ...
identity:
  initialClaimName: ..
  ...
spring:
  profiles:
    active: ...
```